Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -82,14 +83,23 @@ private void RecalculateLayout(double containerWidth)
             double desiredWidth = double.IsNaN(DesiredWidth) ? containerWidth : DesiredWidth;
 
             _columns = CalculateColumns(containerWidth, desiredWidth);
+            bool fillsAtLeastOneRow = true;
 
             // If there's less items than there's columns, reduce the column count;
             if (Items != null && Items.Count > 0 && Items.Count < _columns)
             {
                 _columns = Items.Count;
+                fillsAtLeastOneRow = false;
             }
 
-            ItemWidth = (containerWidth / _columns) - 5;
+            if (fillsAtLeastOneRow || !MaintainAspectRatio)
```

Comment:
Revert these lines in favor of the if check proposed above. This will keep the "stretchy" behavior of the items when there is only one row and maintains the same experience as when there are multiple rows